common/persistence/sql/sqlConfigStore_test.go

```diff
@@ -0,0 +1,177 @@
+// Modifications Copyright (c) 2020 Uber Technologies Inc.
```

Comment:
let's avoid camelCase file names. rename to `sql_config_store_test.go`. Also recommend renaming all files in this folder to keep consistent.